tests/test_contentindexparser.py

```diff
@@ -673,21 +673,21 @@ def check_example1(self, ci_parser):
 
     def test_example1_csv(self):
         # Same test as test_generate_flows but with csvs
-        sheet_reader = CSVSheetReader(self.input_dir / "content_index.csv")
+        sheet_reader = CSVSheetReader(self.input_dir / "workbook_1", "content_index")
         ci_parser = ContentIndexParser(sheet_reader, "tests.input.example1.nestedmodel")
         self.check_example1(ci_parser)
 
     def test_example1_csv_composite(self):
         # Same test as test_generate_flows but with csvs
-        sheet_reader = CSVSheetReader(self.input_dir / "content_index.csv")
+        sheet_reader = CSVSheetReader(self.input_dir / "workbook_1", "content_index")
         reader = CompositeSheetReader([sheet_reader])
         ci_parser = ContentIndexParser(reader, "tests.input.example1.nestedmodel")
         self.check_example1(ci_parser)
 
     def test_example1_split_csv(self):
         # Same test as test_generate_flows but with csvs
-        sheet_reader1 = CSVSheetReader(self.input_dir / "content_index1.csv")
-        sheet_reader2 = CSVSheetReader(self.input_dir / "content_index2.csv")
+        sheet_reader1 = CSVSheetReader(self.input_dir / "workbook_2", "content_index")
```

Comment:
This was an example of multiple content indices in the same folder, and this test cases didn't yield any duplicate warnings. Ideally this test should remain this way (i.e. two content indices in the same folder), because it models a legitimate use case: The user may pass multiple main csv sheets to the commandline tool, which will create CSVReaders with the same path, and cause duplicate warnings. So you'll need to take care of this in `create_flows` in `converters.py`

Reply:
I think we need to say this is not a legitimate case anymore. It's not possible to reference a single sheet in an XLSX or Google Sheet on the command line, so I don't think we should allow it for CSVs.

The "CSV workbook" concept makes it unnecessary to reference individual CSV files on the command line - just reference a single directory and we'll read all the CSVs there. My understanding is that you need at least two sheets to make anything work - a content index plus another - so this should make things easier on the command line.

Therefore, I'm going to delete this test because I think it's not necessary. I'll update the README. The pipeline will have to be updated. Are there any other use cases I should be aware of?